Apps/Examples/Examples/SiwftUI Examples/Util/CameraDebugView.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct CameraDebugView: View {
+    @Binding var camera: CameraState
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            let latlon = String(format: "%.4f, %.4f", camera.center.latitude, camera.center.longitude)
+            Text("\(latlon)")
+            VStack(alignment: .leading) {
+                Text("zoom: \(String(format: "%.2f", camera.zoom))")
+                Text("bearing: \(String(format: "%.2f", camera.bearing))")
+                Text("pitch: \(String(format: "%.2f", camera.pitch))")
+            }.foregroundColor(.gray)
+        }
+        .font(.footnote.safeMonospaced())
+        .padding(5)
+        .floating(RoundedRectangle(cornerSize: CGSize(width: 8, height: 8)))
+    }
+}
+
+@available(iOS 14.0, *)
+struct CameraDebugViewModifier: ViewModifier {
+    var alignment: Alignment
+    @Binding var camera: CameraState
+    func body(content: Content) -> some View {
+        content.safeOverlay(alignment: alignment) {
+            CameraDebugView(camera: $camera)
+        }
+    }
+}
+
+@available(iOS 14.0, *)
+extension View {
```

Comment:
imho we should constraint the type to MapView here:
1) It doesn't make much sense for other views and we don't want to clog up autocomplete.
2) We won't need to pass camera binding to this method, we can grab it from the type itself and it makes more sense to me.

While I was writing this I realized it's part of the sample, so not too important, but still...